apps/penxle.com/src/lib/tiptap/node-views/access-barrier/Component.svelte

```diff
@@ -40,74 +50,150 @@
+        }),
+      }),
+    ],
+    onSubmit: (values) => {
+      updateAttributes({ price: values.price });
+      priceOpen = false;
+    },
+  });
 
-    updateAttributes?.({ price });
-  };
+  $: if (priceOpen) {
+    // eslint-disable-next-line unicorn/prefer-top-level-await
+    void tick().then(() => {
+      update();
+      priceInputEl?.focus();
+    });
+  }
+
+  afterNavigate(() => {
+    priceOpen = false;
+  });
 </script>
 
 {#if editor?.isEditable}
-  <NodeView class="my-4 flex center gap-4 border border-secondary rounded-xl p-4" data-drag-handle>
-    <div class="flex grow gap-4">
-      <i class="i-lc-circle-dollar-sign square-6" />
-
-      <div class="flex grow flex-col">
-        <div class="body-16-b">결제 상자를 옮겨 유료분량을 만들어 보세요</div>
-        <div class="body-13-m text-secondary mt-1">이 상자의 이후 지점부터 결제를 통해 감상할 수 있어요</div>
-
-        <div class="flex gap-2 items-center mt-4">
-          <TextInput
-            class="bg-primary pl-4 py-2 body-16-b rounded-lg"
-            inputmode="numeric"
-            value={price?.toLocaleString()}
-            on:input={handlePriceChange}
-          >
-            <span slot="right-label" class="pr-4 bodylong-16-m text-disabled">포인트</span>
-          </TextInput>
-        </div>
-      </div>
+  <NodeView
+    class={clsx('flex justify-end relative py-8px', selected && 'ring-2 ring-teal-500 rounded-6px')}
+    data-drag-handle
+    draggable
+  >
+    <div class="absolute inset-0 flex center">
+      <div class="line w-full h-1.5px text-gray-300"></div>
     </div>
 
-    <i class="i-lc-grip-vertical square-5 color-icon-secondary" />
-    <button
-      class="square-8 flex center rounded transition duration-300 hover:bg-gray-5 [&>i]:hover:color-gray-50"
-      type="button"
-      on:click={() => (open = true)}
+    <div
+      class="relative rounded-6px p-8px border border-gray-200 flex gap-8px center w-fit mr-20px bg-white text-gray-600 shadow-[0_2px_10px_0] shadow-black/4"
     >
-      <i class="i-lc-trash-2 square-5 color-icon-secondary transition duration-300" />
-    </button>
-  </NodeView>
-
-  <Modal bind:open>
-    <svelte:fragment slot="title">정말 결제상자를 삭제하시겠어요?</svelte:fragment>
+      {#if isLastChild}
+        <button
+          class="text-14px font-medium px-6px py-4px rounded-4px transition hover:bg-gray-100"
+          type="button"
+          on:click={() =>
+            editor
+              ?.chain()
+              .insertContentAt(getPos() + node.nodeSize, { type: 'paragraph' })
+              .focus()
+              .run()}
+        >
+          여기서부터 유료 분량 만들기
+        </button>
+      {:else}
+        <div class="text-14px font-medium px-6px py-4px">여기서부터 유료 분량</div>
+        <div class="w-1px h-14px bg-gray-200" />
+        <button
+          class={clsx(
+            'flex center gap-4px text-14-m px-6px py-4px rounded-4px transition hover:bg-gray-100',
+            priceOpen && 'bg-gray-100',
```

Comment:
```suggestion
            'flex center gap-4px text-14-m px-6px py-4px rounded-4px transition hover:bg-gray-100 aria-pressed:bg-gray-100
          aria-pressed={priceOpen}
```
P3: aria state 로 스타일링할 수 있는 상황에는 적용하는 것이 스타일 코드가 더 간결해진다고 생각합니다.